src/patternfly/components/TreeView/tree-view.scss

```
@@ -36,60 +47,322 @@ $pf-c-tree-view--MaxNesting: 10;
   --pf-c-tree-view__action--hover--Color: var(--pf-global--icon--Color--dark);
   --pf-c-tree-view__action--focus--Color: var(--pf-global--icon--Color--dark);
 
+  // Guides
+  // ================================================================== //
+  --pf-c-tree-view--m-guides--border--Left: var(--pf-c-tree-view--m-guides--base-border--Left); // starting value, this gets updated for each nesting level
+  --pf-c-tree-view--m-guides--border--ZIndex: var(--pf-c-tree-view--m-guides--base-border--ZIndex);
+
+  // Base
+  --pf-c-tree-view--m-guides--base-border--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-tree-view--m-guides--base-border--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-tree-view--m-guides--base-border--Left: calc(var(--pf-c-tree-view__node--PaddingLeft) - var(--pf-c-tree-view--m-guides__list-node--base--BorderWidth));
+  --pf-c-tree-view--m-guides--base-border--Left--offset: calc(var(--pf-c-tree-view__list-item__list-item__node-toggle--Left) + var(--pf-c-tree-view__node-toggle-icon--MinWidth) / 2); // based on toggle positioning
+  --pf-c-tree-view--m-guides--base-border--ZIndex: var(--pf-global--ZIndex--sm);
+  --pf-c-tree-view--m-guides__list-node--base--BorderWidth: var(--pf-global--spacer--lg);
+
+  // List item
+  --pf-c-tree-view--m-guides__list-item--before--Top: 0;
+  --pf-c-tree-view--m-guides__list-item--before--Width: var(--pf-c-tree-view--m-guides--base-border--BorderWidth);
+  --pf-c-tree-view--m-guides__list-item--before--Height: 100%;
+  --pf-c-tree-view--m-guides__list-item--before--BackgroundColor: var(--pf-c-tree-view--m-guides--base-border--BorderColor);
+  --pf-c-tree-view--m-guides__list-item--last-child--before--Top: var(--pf-c-tree-view--m-guides__node--before--Top);
+  --pf-c-tree-view--m-guides__list-item--last-child--before--Height: var(--pf-c-tree-view--m-guides__list-item--last-child--before--Top);
+
+  // Node
+  --pf-c-tree-view--m-guides__node--before--Width: #{pf-size-prem(16px)};
+  --pf-c-tree-view--m-guides__node--before--Height: var(--pf-c-tree-view--m-guides--base-border--BorderWidth);
+  --pf-c-tree-view--m-guides__node--before--Top: #{pf-size-prem(18px)};
+  --pf-c-tree-view--m-guides__node--before--BackgroundColor: var(--pf-c-tree-view--m-guides--base-border--BorderColor);
+
+  // Compact
+  // ================================================================== //
+  // Base
+  --pf-c-tree-view--m-compact--base-border--Left--offset: var(--pf-global--spacer--md);
+  --pf-c-tree-view--m-compact--base-border--Left: calc(var(--pf-c-tree-view__node--PaddingLeft) - var(--pf-c-tree-view--m-compact--base-border--Left--offset));
+  --pf-c-tree-view--m-compact__node--indent--base: var(--pf-c-tree-view__node--indent--base); // based off of the expected width of the toggle button
+  --pf-c-tree-view--m-compact__node--nested-indent--base: var(--pf-global--spacer--lg);
+  --pf-c-tree-view--m-compact--border--Left: var(--pf-c-tree-view--m-compact--base-border--Left);
```

should this be guide?

```suggestion
  --pf-c-tree-view--m-compact--guide--Left: var(--pf-c-tree-view--m-compact--base-border--Left);
```